Makefile

```diff
@@ -91,5 +91,5 @@ release-cli:
 release: release-agent release-server
 
 install:
```

Comment:
I think we could delete install target!

Reply:
removed it and renamed binaries to match new folder path